gqlgen.yml

```diff
@@ -34,6 +34,8 @@ models:
   Task:
     model: github.com/evergreen-ci/evergreen/rest/model.APITask
     fields:
+      failedTestCount:
+        resolver: true
```

Comment:
i don't think forcing a resolver is necessary for fields that are not present on the model. a resolver must exist for fields not present in model; therefore it will be created anyways; therefore forcing a resolver here is redundant.

Reply:
good point